libraries/core_mqtt_demo_dependencies.cmake

```diff
@@ -29,7 +29,8 @@ afr_module_sources(
         ${MQTT_SERIALIZER_SOURCES}
         # List of files added to the target so that these are available
         # in code downloaded from the FreeRTOS console.
-        ${CMAKE_CURRENT_LIST_DIR}/core_mqtt.cmake
+        ${CMAKE_CURRENT_LIST_DIR}/coreMQTT/mqttFilePaths.cmake
+        ${CMAKE_CURRENT_LIST_DIR}/core_mqtt_demo_dependencies.cmake
```

Comment:
Shouldn't `core_mqtt_demo_dependencies.cmake` be part of `core_mqtt_demo_dependencies` module?

Reply:
Yes, moved it toward the bottom of the file for better readability. 
To the metadata logic, the location of the metadata functions does not matter. The matching with the module is done based on the name of the file (thus, kept the file name as `core_mqtt_demo_dependencies.cmake`)